packages/utilities/gel-foundations/src/scripts/bengali-noto-serif.js

```diff
@@ -0,0 +1,214 @@
+const bengaliNotoSerifTypography = {
+  atlas: {
+    groupA: {
+      fontSize: '78',
+      lineHeight: '84',
+    },
+    groupB: {
+      fontSize: '96',
+      lineHeight: '104',
+    },
+    groupD: {
+      fontSize: '140',
+      lineHeight: '148',
+    },
+  },
+  elephant: {
+    groupA: {
+      fontSize: '60',
+      lineHeight: '64',
+    },
+    groupB: {
+      fontSize: '78',
+      lineHeight: '84',
+    },
+    groupD: {
+      fontSize: '116',
+      lineHeight: '124',
+    },
+  },
+  imperial: {
+    groupA: {
+      fontSize: '50',
+      lineHeight: '54',
+    },
+    groupB: {
+      fontSize: '58',
```

Comment:
Some inconguency in grouB and groupD for imperial, royal, foolscap.
Following the previous version: https://github.com/bbc/psammead/blob/latest/packages/utilities/gel-foundations/src/scripts/bengali.js
Am I comparing them with the wrong source?

Reply:
never mind, my bad, I found the source where the values are

Reply:
Hey, we added some additional changes for Imperial, Royal and Foolscap from UX which aren't on the original ticket. I can add it in as a comment to avoid any confusion 👍

Reply:
Yes please, I am in a meeting right now, so I will review them later 👌